server/vcs/markdown/template_resolver.go

```diff
@@ -252,7 +258,16 @@ func (t *TemplateResolver) getApplyTmpl(templateOverrides map[string]string, num
 	return template.Must(template.New("").Funcs(sprig.TxtFuncMap()).Parse(multiProjectApplyTmpl))
 }
 
-func (t *TemplateResolver) getVersionTmpl(templateOverrides map[string]string, common commonData, numPrjResults int, numVersionSuccesses int) *template.Template {
+func (t *TemplateResolver) getLegacyApplyTmpl(templateOverrides map[string]string) *template.Template {
+	if fileName, ok := templateOverrides["apply-legacy"]; ok {
```

Comment:
do we not have a constant for `apply-legacy`? If not lets make one and avoid using string values like this when we can.

Reply:
I have replaced string values with constants where possible.